src/components/Demo/DemoCreate/DemoCreateIssue/index.tsx

```diff
@@ -74,7 +74,9 @@ export const DemoCreateIssue: FunctionComponent = () => {
                 </a>
                 .
               </p>
-              <Button className="bg-cerulean text-white rounded hover:bg-cerulean-500">Test Your File Now</Button>
+              <Link to="/demo/verify">
+                <Button className="bg-cerulean text-white rounded hover:bg-cerulean-500">Test Your File Now</Button>
+              </Link>
```

Comment:
can do this way or, invoke the action Hanlder for the button, depending on whichever is convienient

Reply:
um i think can stick with this, unless there is some critical boo boo that im not aware of